tests/hmm/hmumu_utils.py

```diff
@@ -1542,17 +1544,18 @@ def jet_puid_evaluate(evaluator, era, wp, jet_pt, jet_eta):
     return puid_eff, puid_sf
 
 @numba.njit(parallel=True)
-def compute_eff_product(offsets, jets_mask_passes_id, jets_eff, out_proba):
+def compute_eff_product(offsets, jet_pt, jets_mask_passes_id, jets_eff, out_proba):
     #loop over events in parallel
     for iev in numba.prange(len(offsets)-1):
         p_tot = 1.0
         #loop over jets in event
         for ij in range(offsets[iev], offsets[iev+1]):
-            this_jet_passes = jets_mask_passes_id[ij]
-            if this_jet_passes:
-                p_tot *= jets_eff[ij]
-            else:
-                p_tot *= 1.0 - jets_eff[ij]
+            if jet_pt[ij] < 50:
```

Comment:
Would it be OK to put this cut of 50 GeV somewhere configurable? For example, propagating it from here: https://github.com/jpata/hepaccelerate-cms/blob/master/tests/hmm/pars.py#L499. The issue is that such things hidden deep in the code can easily get forgotton, and it's better to have them in a single place. Thank you!